packages/opentelemetry-metrics/src/Handle.ts

```diff
@@ -34,40 +37,45 @@ export class BaseHandle {
    */
   getTimeSeries(timestamp: types.HrTime): TimeSeries {
     return {
-      labelValues: this._labels.map(value => ({ value })),
+      labelValues: Object.values(this._labelSet.labels).map(value => ({
+        value,
+      })),
       points: [{ value: this._data, timestamp }],
     };
   }
 }
 
 /**
  * CounterHandle allows the SDK to observe/record a single metric event. The
- * value of single handle in the `Counter` associated with specified label
- * values.
+ * value of single handle in the `Counter` associated with specified LabelSet.
  */
 export class CounterHandle extends BaseHandle implements types.CounterHandle {
   constructor(
+    _labelSet: types.LabelSet,
```

Comment:
Since this isn't `private` should this just be named `labelSet`?

Reply:
@draffensperger Yeah that makes sense, then should the `_data` above be named as `data` ? thanks.

Reply:
Well, no `_data` is an actual member of the class. We underscore prefix those so that it's clear in the generated JS that they are private. But here `labelSet` is only a parameter, not a property of the object.